crates/sui-core/src/consensus_handler.rs

```diff
@@ -42,8 +43,8 @@ pub struct ConsensusHandler<T> {
     parent_sync_store: T,
     /// Reputation scores used by consensus adapter that we update, forwarded from consensus
     low_scoring_authorities: Arc<ArcSwap<HashMap<AuthorityName, u64>>>,
-    /// The committee used to do stake computations for deciding set of low scoring authorities
-    committee: Arc<Committee>,
+    /// The narwhal committee used to do stake computations for deciding set of low scoring authorities
+    committee: Committee,
```

Comment:
Why remove the Arc<>?

Reply:
Actually the `Committee` passed here now is the `Narwhal` one , which is created on every epoch and is ephemeral - and we haven't followed across the codebase an approach where we refer to the same `Arc` , so didn't see the upside of maintaining this.